webRepo.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+# if [ $TRAVIS_BRANCH == 'master' ]; then
+  # push new versions of compiled libraries
+  cd /
+  git clone https://github.com/libamtrack/web.git
+  cd /web
+  current_date_time="`date "+%Y%m%d_%H%M%S"`"
+  git checkout -b "feature/compiled_lib/$current_date_time"
+
+  rm -f /web/src/libat.wasm
+  cp /compiled/libat.wasm /web/src/libat.wasm
+
+  rm -f /web/src/static/js/libat.wasm 2>/dev/null || :
+  cp /compiled/libat.wasm /web/src/static/js/libat.wasm
+
+  rm -f /web/src/static/js/libat.js
+  cp /compiled/libat.js /web/src/static/js/libat.js
+
+  rm -f /web/src/static/js/libatwithoutwasm.js
+  cp /compiled/libatwithoutwasm.js /web/src/static/js/libatwithoutwasm.js
+
+  git add -A
+  git commit -m "Update compiled library"
+  git push "https://marwin1991:$GH_TOKEN@github.com/libamtrack/web.git"
+
+
+  # install hub tool and create pull request in web repo
+  cd /
```

Comment:
Is it a lot of work to create PR without the help of external tool like hub ?

Reply:
PR is "not standart" of a git and [GitHub API documentation](https://developer.github.com/v3/pulls/) shows that probbly it will take more work that install hub tool